password.py

```diff
@@ -1,49 +1,302 @@
+        username = input("Enter a Username:\n")
+        if len(username) < 5:
+            print("Username has to be 5 characters or more.")
+            continue
+        else:
+            account_info_dict["Username"] = username
+            break
+
+    if password_randomizer_query() == 'no':
+        while True:
+            password = input("Enter a Password:\n")
+            if len(password) < 6:
+                print("Password has to be 6 characters or more.")
+                continue
+            else:
+                account_info_dict["Password"] = password
+                break
+    
+    account_dict[username] = account_info_dict
+
+    security_questions()
+
+    account_save()
+
+    return 
+
+# Asks user to pick a question and saves the answer
+def security_questions():
+     sec_questions_dict = {
+         "1":"What's your pet's name",
+         "2":"What's your mother's middle name",
+         "3":"What's your best friend's nickname",
+         "4":"Which hospital were you born in"
+      }
+
+     print("-"*26)
+     for key,val in sec_questions_dict.items():
+         print(key,":",val)
+     print("-"*26)
+
+     security_choice = input("Please choose a security question (Type in corresponding number):\n")
+
+     value = sec_questions_dict[security_choice]
+     print(value)
+     question_answer = input("")
+
+     account_info_dict["Security Question"] = value
+     account_info_dict["Security Answer"] = question_answer
+
+     return value, question_answer
 
 # Asks to generate a randomized password
 def password_randomizer_query():
+    accepted_answers = 'yes','no'
+
+    while True:
+        rand_answer = input("Do you want a randomized password? ('yes' or 'no'):\n")
+        if rand_answer == 'yes':
+            gen_pass = random_pass()
+            account_info_dict["Password"] = gen_pass
+            print("Your random pass is: " + gen_pass)
+            break
+        elif rand_answer == 'no':
+            break
+        elif rand_answer != accepted_answers:
+            print("Incorrect use.")
+
+    return rand_answer
 
-    return
 # Generates a randomized password using letters and digits
 def random_pass():
+    string = shane_char.ascii_characters_and_num()
+    letters_and_num = string.letters + string.digits
+    rand_password = random.choice(string.lowercase)
+    rand_password += random.choice(string.uppercase)
+    rand_password += random.choice(string.digits)
+    rand_password += random.choice(string.special_characters)
+
+    for characters in range(6):
+        rand_password += random.choice(letters_and_num)
+
+    password_list = list(rand_password)
+    random.SystemRandom().shuffle(password_list)
+    rand_password = ''.join(password_list)
+
+    return rand_password
 
-    return
 # Saves and stores account
 def account_save():
 
-    return
+    account_file = open('account.txt','w')
+    account_file.write(str(account_dict))
+    account_file.close()
+
+    return account_dict
```

Comment:
We stringify it and write it to a file again, which I'm sure you know doesn't work for if we want to do anything other than print this back the next time this program is called.

As we chatted about you could save this to json, which is relatively easy because the dictionary structure you have going on here, and then your program could read that json back in to "load" it back into memory.